murdock/main.py

```diff
@@ -167,7 +167,7 @@ async def finished_job_delete_handler(
 ):
     query = JobQueryModel(before=before)
     if not (jobs := await murdock.remove_finished_jobs(query)):
-        raise HTTPException(status_code=404, detail="Found no finished job to remove")
+        return []
     return jobs
```

Comment:
Since `murdock.remove_finished_jobs` returns a list of JobModel and we don't have a special case when it's empty, then this whole block could be simplified like:

```python
    return await murdock.remove_finished_jobs(JobQueryModel(before=before))
```

Reply:
yup